packages/run-protocol/src/runStake/runStake.js

```diff
@@ -197,13 +198,19 @@ export const start = async (
     }),
   );
 
-  const creatorFacet = makeGovernorFacet(
-    Far('runStake creator', {
-      provideAttestationMaker: att.creatorFacet.provideAttestationTool,
-    }),
-  );
+  /** @type {ERef<RunStakeCreator>} */
+  const creatorFacet = Far('runStake creator', {
+    provideAttestationMaker: att.creatorFacet.provideAttestationTool,
+    makeCollectFeesInvitation: () => {
+      return makeMakeCollectFeesInvitation(
+        zcf,
+        rewardPoolSeat,
+        debtMint.getIssuerRecord().brand,
+        'RUN',
+      ).makeCollectFeesInvitation();
+    },
+  });
 
-  // @ts-expect-error makeGovernorFacet loses type info
-  return { publicFacet, creatorFacet };
+  return { publicFacet, creatorFacet: makeGovernorFacet(creatorFacet) };
```

Comment:
Does this mean runStake governance was previously busted? Did we not have any tests for it?

Reply:
I just moved the `makeGovernorFacet` down here to make more obvious that there are two slightly different things named `creatorFacet`.